LearnNet_CartingService/Controllers/v0/CartItemsController.cs

```diff
@@ -2,12 +2,12 @@
 using LearnNet_CartingService.Core.DTO;
 using Microsoft.AspNetCore.Mvc;
 using LearnNet_CartingService.Core.Interfaces;
+using Asp.Versioning;
 
-// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-
-namespace LearnNet_CartingService.Controllers
+namespace LearnNet_CartingService.Controllers.v0
```

Comment:
What is the purpose of v0 controller? I thought v1 and v2 would be enough to meet the requirements